sdk/communication/Azure.Communication.PhoneNumbers/tests/SessionRecords/PhoneNumbersClientLiveTests/UpdateCapabilities.json

```diff
@@ -153,7 +153,7 @@
       }
     },
     {
-      "RequestUri": "https://smstestapp.communication.azure.com/phoneNumbers/\u002BSanitized?api-version=2022-12-01",
+      "RequestUri": "https://smstestapp.communication.azure.com/phoneNumbers/Sanitized/Sanitized",
```

Comment:
I don't know why this change is needed, without it the test throws the error:
```
Message: 
Azure.Core.TestFramework.TestRecordingMismatchException : Unable to find a record for the request GET https://smstestapp.communication.azure.com/phoneNumbers/Sanitized/Sanitized
Uri doesn't match:
    request <https://smstestapp.communication.azure.com/phoneNumbers/Sanitized/Sanitized>
    record  <https://smstestapp.communication.azure.com/phoneNumbers/+Sanitized?api-version=2022-12-01>
```
But it doesn't make sense to me that it sanitize twice